packages/core/src/App/Containers/QuestionnaireModal/questionnaire-modal.scss

```diff
@@ -0,0 +1,70 @@
+.dc-modal__container_questionnaire-modal {
```

Comment:
these 3 classnames will affect global styles of:
`.dc-modal__container_questionnaire-modal`
`.dc-modal-body:first-child`
`#modal_root`

Let's use another approach to set styles in your modal.

Please check what classnames the <Modal component accepts as props and pass them instead of overriding global modal styles.
Please make sure that the styles you add are incapsulated inside `.questionnaire-modal {`

Reply:
I see now, thanks! 🙏

Reply:
actually business requirements changed and I can just remove this modifiers 😄

Reply:
Business decided to made another way, but I still need modify `#modal_root`
It'll replace styles globally, but this is a last phase before `enableApp` and redirection to tradershub

I believe that we can proceed with that implementation, since it's difficult to refactor/or find a place to fix

Reply:
finally resolved by javascript code in useEffect on mount and unmount lifecycle

Reply:
@NikitK-deriv there should not be a need for changing the color. let's make it cleaner by making the empty dialog (modal) behind your modal closed/not open at all. Is that empty dialog still there? :)